doc/tutorials/core/mat_the_basic_image_container/mat_the_basic_image_container.markdown

```diff
@@ -192,10 +192,12 @@ object in multiple ways:
 
     ![](images/MatBasicContainerOut3.png)
 
--   For small matrices you may use comma separated initializers:
+-   For small matrices you may use comma separated initializers or initializer lists (C++ 11 support is required in the last case):
```

Comment:
Could you please use "C++11" without space (reference: https://en.wikipedia.org/wiki/C%2B%2B11)
It will be nice to rename `CV_CXX_11` -> `CV_CXX11` (google for `"HAVE_CXX11"` vs `"HAVE_CXX_11"`)